script/Ciel/Day 8/08_01_CIEL08_1B.txt

```diff
@@ -178,7 +178,7 @@ I think simple relationships are useful in their own way.
 [sha:b42136e15a7110c594f0275e63c1eace1d942eb4]{
 -- Page 46, Offset 18377.
 -- 色んなところに行けるようになって、いろんな人と知り合えるようになった以上、人間関係も最適化させないと潰れちまう……ってのは友人からの受け売りだけど」
-Once you've been all sorts of places and met with all sorts of people,
+Once you've been to all sorts of places and met with all sorts of people,
```

Comment:
kinda went back and forth on this one but i think with the "_with_ all sorts" it works.

Reply:
Yeah I didn't love this one either tbh, if it was also weird to you I'd just revert

Reply:
i'm alright with reverting